src/views/udns/list/components/ListCreateButton.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FC } from 'react';
+
+import {
+  ListPageCreateButton,
+  ListPageCreateDropdown,
+  useModal,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { useNetworkingTranslation } from '@utils/hooks/useNetworkingTranslation';
+import {
+  ClusterUserDefinedNetworkModelGroupVersionKind,
+  UserDefinedNetworkModel,
+  UserDefinedNetworkModelGroupVersionKind,
+} from '@utils/models';
+import { getNamespace } from '@utils/resources/shared';
+import { isPrimaryUDN } from '@utils/resources/udns/helper';
+import { ClusterUserDefinedNetworkKind, UserDefinedNetworkKind } from '@utils/resources/udns/types';
+
+import UserDefinedNetworkCreateModal from './UserDefinedNetworkCreateModal';
+
+type ListCreateButtonProps = {
+  allUDNs: Array<ClusterUserDefinedNetworkKind | UserDefinedNetworkKind>;
+  namespace: string;
+};
+
+const ListCreateButton: FC<ListCreateButtonProps> = ({ allUDNs, namespace }) => {
```

Comment:
Since this is a specific component related to UDNs and not a generic component that can be used across multiple CRDs I would rename this to UDNListCreateButton